src/main/java/com/uipath/uipathpackage/models/PackOptions.java

```diff
@@ -39,11 +45,59 @@ public void setOutputType(String outputType) {
         this.outputType = outputType;
     }
 
+    public boolean isSplitOutput() {
+        return splitOutput;
+    }
+
+    public void setSplitOutput(boolean splitOutput) {
+        this.splitOutput = splitOutput;
+    }
+
     public String getVersion() {
         return version;
     }
 
     public void setVersion(String version) {
         this.version = version;
     }
+
+    public String getRepositoryUrl() {
```

Comment:
These new getters seem to be unused in our codebase. Are these getters used during serialization?

Reply:
same as splitOutput